bin/pepProxy

```diff
@@ -36,8 +36,10 @@ function loadConfiguration() {
         'LOG_LEVEL',
         'ACCESS_HOST',
         'ACCESS_PORT',
+        'ACCESS_PROTOCOL',
```

Comment:
I think (but not sure) this is a table in the doc with a list of env variables. Have that list being aligned with the new added variables in the code?